Libplanet/Blockchain/BlockChain.cs

```diff
@@ -172,7 +172,11 @@ IEnumerable<IRenderer<T>> renderers
                 GetBalance,
                 trieGetter);
 
-            if (Count == 0)
+            if (inFork && !(StateStore is IBlockStatesStore))
```

Comment:
It would be better if we left comments about why we query `StateStore`'s type.

Reply:
👌